e2e/setup/authentication.ts

```diff
@@ -114,6 +114,8 @@ setup.describe("authentication against a mock backend", () => {
 
     await page.waitForURL(/session\/3\/join/);
 
+    await page.getByTestId("current-user").click();
```

Comment:
You've simultaneously changed this here and the `Avatar.tsx` `data-testid` to `current-username`... is this desired?